test/library/packages/LinearAlgebra/correctness/testExpm.chpl

```diff
@@ -0,0 +1,158 @@
+use LinearAlgebra;
```

Comment:
This test looks like it is missing an expected output file (see our documentation on the [test system](https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#a-correctness-test).  Since you're using `assertAlmostEqual`, I think it should end up empty, but you'll still need a file.